lesson3/src/HomeWork.java

```diff
@@ -0,0 +1,84 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class HomeWork {
+    public static void main(String[] args) {
+        System.out.println(operation(1));
+        System.out.println(operation(0));
+        foobar(6);
+        foobar(10);
+        foobar(15);
+        printMatrix();
+        printPrimeNumbers();
+    }
+
+    /**
+     * Method should perform some operation with int "number" based on some conditions:
+     *  - if number is positive - increase it by 1
+     *  - if number is negative - decrease it by 2
+     *  - if number is equal to zero - replace it with 10
+     *  return number value after operation
+     */
+    public static int operation(int number) {
+        Scanner scanner = new Scanner(System.in);
+        int sum = scanner.nextInt();
```

Comment:
you don't need to read anything from console. Use int number

Reply:
and how do I know what the number will be? int

Reply:
should I put a number instead of int ?

Reply:
no I meant variable called "number"))